tests/CMakeLists.txt

```diff
@@ -31,13 +31,15 @@ if(${run_unittests})
         add_subdirectory(timer_win32_ut)
         add_subdirectory(interlocked_hl_win32_ut)
         add_subdirectory(interlocked_win32_ut)
+    else()
+        add_subdirectory(interlocked_linux_ut)
```

Comment:
elseif...

Needs to check for C11 support at least I think (that should be sufficient, though it's possible for a compiler to support C11 without atomics, we can deal with that if it were ever necessary).

[https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_C_KNOWN_FEATURES.html#prop_gbl:CMAKE_C_KNOWN_FEATURES](https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_C_KNOWN_FEATURES.html#prop_gbl:CMAKE_C_KNOWN_FEATURES)

I think you can just check it by seeing if `c_std_11` is supported in CMAKE_C_COMPILE_FEATURES.

I think the syntax is:

`elseif (c_std_11 IN_LIST CMAKE_C_COMPILE_FEATURES)`